# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/speakersafetyd/hotfix-install-t.patch
# Copyright (C) 2023 - 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- speakersafetyd-0.1.9/Makefile.vanilla	2024-03-21 17:38:05.006835976 +0100
+++ speakersafetyd-0.1.9/Makefile	2024-03-21 17:38:36.339834397 +0100
@@ -21,7 +21,7 @@
 	install -dDm0755 $(DESTDIR)/$(UDEVDIR)
 	install -pm0644 95-speakersafetyd.rules $(DESTDIR)/$(UDEVDIR)/95-speakersafetyd.rules
 	install -dDm0755 $(DESTDIR)/$(SHAREDIR)/speakersafetyd/apple
-	install -pm0644 -t $(DESTDIR)/$(SHAREDIR)/speakersafetyd/apple $(wildcard conf/apple/*)
+	install -pm0644 $(wildcard conf/apple/*) $(DESTDIR)/$(SHAREDIR)/speakersafetyd/apple
 	install -dDm0755 $(DESTDIR)/$(VARDIR)/lib/speakersafetyd/blackbox
 	install -dDm0755 $(DESTDIR)/$(TMPFILESDIR)
 	install -pm0644 speakersafetyd.tmpfiles $(DESTDIR)/$(TMPFILESDIR)/speakersafetyd.conf
